apptainer/demos/openradioss/README.md

```diff
@@ -0,0 +1,370 @@
+```bash
+gcloud builds submit --config=paraview.yaml .
+```
+
+Go back to the parent directory
+```bash
+cd ..
+```
+
+### Vortex-Radioss
+
+Build Vortex-Radioss container.
+
+```bash
+gcloud builds submit --config=vortex-radioss.yaml .
+```
+
+Go back to the parent directory
+```bash
+cd ..
+```
+
+
+## HPC System Deployment
+
+Use the [OpenRadioss blueprint](./hpc/openradioss-demo.yaml) in the `hpc` directory as the basis for your HPC system. It is conifgured to deploy two partitions. 
+- A `compute` partition that will dynamically allocate up to 10 [compute-optimized](https://cloud.google.com/compute/docs/compute-optimized-machines#c2_machine_types) C2-Standard-60 instances.
+- A `gpu` that will dynamically allocate up to 4 [general-purpose](https://cloud.google.com/compute/docs/general-purpose-machines#n1_machines) N1-STANDARD-8 instances.
+
+Change directories to the `hpc` directory
+```bash
+cd hpc
+```
+
+Now you can create the deployment artifacts with the command
+
+```bash
+gcluster create openradioss-demo.yaml --vars project_id=$(gcloud config get core/project) --skip-validators=test_tf_version_for_slurm
+```
+
+You should see output that looks like
+
+```
+To deploy your infrastructure run:
+
+glucster deploy openradiossdemo
+
+Find instructions for cleanly destroying infrastructure and advanced manual
+deployment instructions at:
+
+openradiossdemo/instructions.txt
+```
+
+Per the instructions, bring up your HPC system with the command
+```bash
+gcluster deploy openradiossdemo --auto-approve
+```
+
+Go back to the parent directory
+```bash
+cd ..
+```
+
+## Login node
+
+SSH via IAP tunnel with the command
+```bash
+gcloud compute ssh \
+  $(gcloud compute instances list --filter="NAME ~ login" --format="value(NAME)") \
+  --tunnel-through-iap
+```
+
+## Simulation
+
+This demonstration uses one of the models from OpenRadioss official website(https://openradioss.org/models/).
+
+Download Yaris Impact Model in LS-DYNA® format from CCSA and OpenRadioss launch file.
+```bash
+wget -q https://media.ccsa.gmu.edu/model/2010-toyota-yaris-detailed-v2j.zip
+unzip 2010-toyota-yaris-detailed-v2j.zip
+cd 2010-toyota-yaris-detailed-v2j
+wget -q -O YarisOpenRadioss.key https://openradioss.atlassian.net/wiki/download/attachments/30539777/YarisOpenRadioss.key?api=v2
+```
+
+Go back to the parent directory
+```bash
+cd ..
+```
+
+Now create a `job.sh` file that specifies the compute environment for the simulation and all the tasks to be carried out as part of the simultion.
+
+```bash
+cat << EOF > job.sh
+#!/bin/bash
+#SBATCH --partition=compute
+#SBATCH --nodes=3
+#SBATCH --ntasks-per-node=30
+
+cd ~/2010-toyota-yaris-detailed-v2j
+
+openradioss starter_linux64_gf -i YarisOpenRadioss.key -np 90
+
+export PMIX_MCA_gds=hash
+srun --mpi=pmix apptainer run --sharens ~/bin/openradioss engine_linux64_gf_ompi -i YarisOpenRadioss_0001.rad
+EOF
+```
+
+Set up access to the Artifact Registry repository
```

Comment:
It is important to add this environment variable, and use it in the subsequent `apptainer registry login ...` command otherwise that command will fail.